test/factories/genres.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :genre do
+    name { "MyString" }
+    description { "MyText" }
```

Comment:
description probably needs to be longer in order to pass validation

Reply:
Adjusted description field to pass validation